package.json

```diff
@@ -3540,6 +3540,12 @@
           "description": "%cmake-tools.configuration.cmake.pinnedCommands.description%",
           "default": [],
           "scope": "resource"
+        },
+        "cmake.enableKitScan": {
```

Comment:
We might consider using `cmake.enableAutomaticKitScan`, since kit scanning in general is still supported, we're just disabling the automatic kit scan. (Right?)